wagtail/admin/templates/wagtailadmin/pages/create.html

```diff
@@ -13,7 +13,11 @@
         <div class="row row-flush">
             <div class="left col9 header-title">
                 <h1 class="icon icon-doc-empty-inverse">
-                {% trans 'New' %} <span>{{ content_type.model_class.get_verbose_name }}</span></h1>
+                    {% trans 'New' %} <span>{{ content_type.model_class.get_verbose_name }}</span>
```

Comment:
My concern here is for screen readers, though I'm not sure what the solution is. I tested this briefly in Orca in Firefox, and found that the small tag did not cause a break - there's no separation between the heading and subheading. While this isn't a huge issue, it could lead to long and confusing headings for screen reader users. Perhaps this would be better outside the h1 to separate it, or by using something else to denote the heading/subheading divide?

Reply:
Semantically, the description does not belong in the heading, so 👍for moving it outside.
This may also move the description under "New/Editing <Page type>" making it cleaner.

Reply:
I would suggest moving it out because I presume the text could get relatively lengthy for real-world descriptions.

If you do want to keep it in, you can add a screen-reader-only full stop `<span class="visuallyhidden">.</span>` in-between the title and description so there is a pause when announcing the text. In that case it would be nice for the `<title>` and `<h1>` to match.